content/docs/principles/proudly-invented-elsewhere.md

```diff
@@ -6,3 +6,9 @@ description: Software development is not new. That means somebody might have alr
 # Proudly Invented Elsewhere
 
 Software development is not new. That means somebody might have already faced a similar problem and has solved it. Use libraries whenever you can. Even though a library does 70-80% of your requirement, consider using it.
+
+We prefer using established, contributed libraries over writing custom code. This approach is more cost-effective and requires fewer programming skills for maintenance. These projects often have established best practices and allow us to standardize our processes. To implement this, we download and configure relevant libraries. We choose to use these libraries when their functions closely align with our requirements, anticipated function changes can be accommodated through configuration, and other users should be able to make configuration changes.
```

Comment:
Is there a better word for this than "contributed libraries"? Even "open-source" is specific because we could be okay with even proprietary libraries written and maintained by others.

Reply:
@hussainweb What about just "libraries"?

Reply:
Yeah, sounds better.